doc/odoc_for_authors.mld

```diff
@@ -294,6 +294,42 @@ doc.
 *)
 v}
 
+{4:language_headers Language Headers}
+
+As of [odoc.2.1] it is possible to write blocks with explicit language headers,
+similar to markdown code blocks:
```

Comment:
I don't think it's useful to compare with markdown. I think 3 examples is too much for a relatively common feature.

Reply:
I think that's reasonable. I'll modify it.